en/config/actuators.md

```diff
@@ -140,32 +140,60 @@ Note that you will need to also ensure that the ESC associated with bidirectiona
 The control surfaces section of the geometry panel lets you set the number and types of control surfaces that are present on the vehicle.
 You may also need to set trim and slew rate values in some cases.
 More advanced users can also configure the roll scale, yaw scale, and pitch scale (generally the defaults are acceptable, and this is not needed).
-
 An "example" control surface section for a vehicle with two ailerons is shown below.
 Note that ailerons only affect roll, so the pitch and yaw fields are disabled.
 
 ![Control Surface Setup Example](../../assets/config/actuators/control_surfaces_geometry.png)
```

Comment:
I updated this image (see above) to show both the advanced and basic cases. 
1. I couldn't get it to show lock control surfaces in hover - is that only for VTOL? What frame would I use to show this?
2. FYI the daily QGC hides any fields not needed by any of the displayed surfaces - so we'll have to update the images when daily becomes a release, and some of the text.

Reply:
1. Yes that's VTOL only. eg `make px4_sitl_default gazebo_standard_vtol`
2. okay :+1:

Reply:
Thanks. Rather than regenerate this now, I added a "VTOL only" note like this:
![image](https://user-images.githubusercontent.com/5368500/232977895-2ec483bc-82b1-47fc-8174-d72865d7f293.png)